ev3dev2/motor.py

```diff
@@ -1803,6 +1919,9 @@ def on_for_seconds(self, left_speed, right_speed, seconds, brake=True, block=Tru
         self.right_motor.time_sp = int(seconds * 1000)
         self.right_motor._set_brake(brake)
 
+        log.debug("%s: on_for_seconds %ss at left-speed %s, right-speed %s" %
```

Comment:
If we're using debug logging more widely can you add some info somewhere (README? Docs?) saying how to enable the output?